Orange/widgets/data/owtranspose.py

```diff
@@ -38,6 +38,9 @@ class Warning(OWWidget.Warning):
         duplicate_names = Msg("Values are not unique.\nTo avoid multiple "
                               "features with the same name, values \nof "
                               "'{}' have been augmented with indices.")
+        discrete_attrs = Msg("Values of categorical features will change.\n"
+                             "They will be encoded with suitable numeric "
+                             "representation.")
```

Comment:
I would you past tense here. When the user reads the message, the deed has already been done. :)

Reply:
That is true, does the present passive voice sounds okay? 

Values of categorical features _are changed_. They _are encoded_ with suitable numeric representation.

Reply:
It is not entirely clear what "suitable" means. Perhaps "They will be encoded as numbers."

Alternative: The widget is so narrow that the message won't fit anyway, so it doesn't hurt if it's too long: we can write just "Categorical features have been encoded as numbers." Without any `\n` and detils.